packages/babel-generator/src/node/parentheses.js

```diff
@@ -2,6 +2,7 @@ import * as t from "@babel/types";
 
 const PRECEDENCE = {
   "||": 0,
```

Comment:
While it's unobservable now, `??` should actually be the lowest precedence of all tokens. Meaning we'll need to increase every other token's precedence.

Reply:
Ok, so I am assigning a precedence of 0 to `??` and will increment that of others by 1. BTW, I am still not clear as to how to get to know the precedence that we follow here.

Reply:
Oh, actually, you're right. `SortCircuitExpression` is both `CoalesceExpression` and `LogicalORExpression`, so they have the same precedence. I was remember when we had the spec using grammar flags, and there it was `CoalesceExpression -> LogicalORExpression`.

Reply:
Oh, I had already made the changes. 😅 Reverting them.